### PR TITLE
More study documentation.

### PR DESCRIPTION
**Pull request by Gundersanne**
Some doc pages in development/misc:

documentation on apex-experiments gerrit

documentation on apex-experiments gerrit

### CODE REVIEW

**Comment by Gundersanne** [comment]
pushing experiments and pulling results:

image

File: studies/index.md [hunk 1]
```diff
@@ -70,6 +70,67 @@ devices) on the studies page.
...
+# under projects.
+git clone <url>
+cd <repository>
+# First create a commit.
+git add <experiment files and stimuli>
+git commit
+
+# Push to the experiment branch.
+git push origin HEAD:refs/heads/experiments-1
+# If you want to do a gerrit review first, push to refs/for.
```
Comment (tomfrancart) [inline studies/index.md]: I'd remove this, it complicates things

File: studies/index.md [hunk 2]
```diff
@@ -70,6 +70,67 @@ devices) on the studies page.
...
+git read-tree $BRANCHES
+# Commit them into collected-results.
+git commit -m "Merge $BRANCHES into collected-results."
+# Reset index.
+git reset --hard
+```
+
+The results should now be collected on the `collected-results` branch, and
+present in your working directory.
+
```
Comment (tomfrancart) [inline studies/index.md]: In a typical workflow, you'd want to do this a number of times when new results come in. What would be the best strategy to deal with that? Start over? Or merge into the existing one?

File: studies/index.md [hunk 3]
```diff
@@ -70,6 +70,67 @@ devices) on the studies page.
...
+# First create a commit.
+git add <experiment files and stimuli>
+git commit
+
+# Push to the experiment branch.
+git push origin HEAD:refs/heads/experiments-1
+# If you want to do a gerrit review first, push to refs/for.
+git push origin HEAD:refs/for/experiments-1
+```
+
```
Comment (tomfrancart) [inline studies/index.md]: Does there need to be a flow runner present with a certain name?

File: studies/index.md [hunk 4]
```diff
@@ -70,6 +70,67 @@ devices) on the studies page.
...
+### fetching results
+
+Fetching the results is somewhat complex because the result branches don't have
+any shared history.
+
```
Comment (tomfrancart) [inline studies/index.md]: Explain how the results branches are named?

**Pull request merged**